.github/workflows/integrate.yaml

```diff
@@ -43,6 +43,14 @@ jobs:
     - name: Run unit tests
       run: tox -e unit
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    with:
+      charm-path: .
+      model: kubeflow
```

Comment:
envoy can be deployed in any namespace, I suggest we use the same one as integration tests workflow i.e. `testing`

Reply:
Makes sense!